### PR TITLE
[BP-29] Doc changing validators on IBFT network

### PR DESCRIPTION
**Pull request by sgregglives**
Signed-off-by: Steven Gregg <steven.gregg@consensys.net>

## PR description
Document how to change validators on IBFT Network

## Fixed Issue(s)

BP-29

### CODE REVIEW

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md [hunk 1]
```diff
@@ -165,15 +169,49 @@ To discard your proposal after confirming the validator was added, call `ibft_di
...
+
+In some cases, such as when the results of voting would not yield a majority, you can bypass voting and manually change validators in your genesis file.
+
+To add or remove validators without voting:
+
+1. Stop the network.
+1. Copy the `transitions` object and paste it after the `config` object in your genesis file. Specify your own values, where
+
+    * `<BlockNumber>` is the upcoming block where you want the change in validators to occur.
+    * `<AValidatorAddress>` is the account address for a validator to be added.
```
Comment (NicolasMassart) [inline docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: Why the initial A in Validator address ?

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md [hunk 2]
```diff
@@ -165,15 +169,49 @@ To discard your proposal after confirming the validator was added, call `ibft_di
...
+
+    !!! example "Transitions Object Example"
```
Comment (NicolasMassart) [inline docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: This is more the syntax, the example would use real values. You can have both in the same code block by using tabs

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md [hunk 3]
```diff
@@ -165,15 +169,49 @@ To discard your proposal after confirming the validator was added, call `ibft_di
...
+          "<AValidatorAddress>",
+          "<AValidatorAddress>"
+        ]
+      }
+    ]
+    }
+    },
+    ``` 
+
+1. Restart the network.
```
Comment (NicolasMassart) [inline docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: All the nodes or only the node for which you changed the genesis?

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md (hunk 1 above)
Comment (sgregglives) [reply docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: Updated.

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md (hunk 2 above)
Comment (sgregglives) [reply docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: Added this formatting.

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md (hunk 3 above)
Comment (sgregglives) [reply docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: We're restarting the node after the genesis file has been updated with validators, whether added or removed.

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md [hunk 4]
```diff
@@ -165,15 +170,87 @@ To discard your proposal after confirming the validator was added, call `ibft_di
...
+    Do not specify a transition block in the past. Specifying a transition block in the past may result in 
+    unexpected behaviour.  
+
+To add or remove validators without voting:
+
+1. Stop all nodes in the network.
+1. In the genesis file, add the `transitions` configuration item where: 
+
+    * `<BlockNumber>` is the upcoming block at which to change validators.
+    * `<ValidatorAddressX> ... <ValidatorAddressZ>` are the account addresses of the validators after `<BlockNumber>`.
```
Comment (rain-on) [inline docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: nit: is it worth mentioning they are _string_ elements representing the account addresses (i.e. define the json type)

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md [hunk 5]
```diff
@@ -165,15 +169,64 @@ To discard your proposal after confirming the validator was added, call `ibft_di
...
 
-### Epoch Transition
+#### Epoch Transition
 
 At each epoch transition, all pending votes collected from received blocks are discarded. Existing proposals remain 
 in effect and validators re-add their vote the next time they create a block. 
 
 An epoch transition occurs every `epochLength` blocks where `epochlength` is defined in the IBFT genesis file.
 
+### Adding and Removing Validators without Voting
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: Leaving as is as discussed.

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md [hunk 6]
```diff
@@ -165,15 +169,64 @@ To discard your proposal after confirming the validator was added, call `ibft_di
...
+
+In some cases, such as when the results of voting would not yield a majority, you can bypass voting and manually change validators in your genesis file.
+
+To add or remove validators without voting:
+
+1. Stop the network.
+1. Copy the `transitions` object and paste it after the `config` object in your genesis file. Specify your own values, where
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: Has been updated

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md (hunk 4 above)
Comment (MadelineMurray) [reply docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: Probably.  Have updated.

Comment (MadelineMurray) [reply docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: Updated

**Pull request merged**